ui/raidboss/data/04-sb/trial/shinryu.txt

```diff
@@ -0,0 +1,84 @@
+hideall "--Reset--"
+hideall "--sync--"
+
+# ii 1F8B 1F8D 1F8E 1F8F 1F90 1F94 1F9A 1F9C 1F9E 1FA3 1FBB 1FCE 1FCF 1FF4 2218 2219 221A 244B 24F0
+
+# Elemental Attack is one of:
+# Aerial Blast, Judgment Bolt, Diamond Dust, Hellfire, Earthen Fury, Tidal Wave
+
+0.0 "--Reset--" sync / 21:........:40000010:/ window 10000 jump 0
+
+0 "Start" sync /Engage!/ window 0,1
+1.0 "--sync--" sync /:(Left Wing|Right Wing):1FA9:/ window 1,2
+20.6 "Tidal Wave" sync /:Shinryu:1FAA:/ window 20.6,5
+38.8 "Hypernova/Levinbolt" sync /:Right Wing:1F9[9B]:/
+49.9 "Akh Morn" sync /:Shinryu:1FA4:/
+59.2 "Summon Icicle" sync /:Left Wing:1F9F:/
+59.9 "Icicle Impact x2" # sync /:Icicle:1FA0:/
+64.4 "Spikesicle x2" # sync /:Icicle:1FA1:/
+75.2 "Ice Storm" sync /:Left Wing:1FA2:/
+88.3 "Earth Breath" sync /:Shinryu:1F9D:/ # May be skipped with high DPS
+115.8 "Hellfire/Judgment Bolt"sync /:Shinryu:1FA[BC]:/ window 30,30
+118.1 "--untargetable--"
+
+# Intermission.
+# The given times for Meteor Impact are the latest they can occur.
```

Comment:
Are there start casting times for those that you could sync to instead?

Reply:
It looks like it? I'm having trouble getting that to sync, just trying a couple more things here...

Reply:
That seems to work. I wish there were a good way to include those by default at timeline generation, but it's not a simple thing to filter like `targetable/untargetable`.

Reply:
Yeah, I think this is harder.  I mean, one option would be to have a timeline option where you could pass a flag and it would always print them, and then you could use that and splice them in? It doesn't get needed all that many times, I think, so it's not a huge missing feature either.